code_style.sh

```diff
@@ -60,7 +60,7 @@ fi
 
 # format C/C++
 clang_format_args="$([ -n "$fix" ] && echo '-i' || echo '--dry-run --Werror')"
-if ! find_cfiles -exec "${clang_format_exe}" $clang_format_args '{}' '+'; then
+if ! find_cfiles -exec "${clang_format_exe}" $clang_format_args --verbose '{}' '+'; then
```

Comment:
Thank you for your PR, but could you kindly split it and make this change in a PR of its own? Then it will be reviewable and approvable by the most appropriate persons.

Reply:
I reverted this change.